scripts/runningnotes_from_sequencing.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+DESC = """EPP script to summarize sequencing start results to the projects running notes
+"""
+import datetime
+import re
+import xml.etree.ElementTree as ET
+from argparse import ArgumentParser
+
+from genologics.config import BASEURI, PASSWORD, USERNAME
+from genologics.entities import Process
+from genologics.lims import Lims
+from write_notes_to_couchdb import write_note_to_couch
+
+from scilifelab_epps.wrapper import epp_decorator
+
+regex_project_line = re.compile(r"^\[(P\d+)\][\s]*:")
+TIMESTAMP: str = datetime.datetime.now().strftime("%y%m%d_%H%M%S")
+
+
+@epp_decorator(script_path=__file__, timestamp=TIMESTAMP)
+def main(args):
+    lims = Lims(BASEURI, USERNAME, PASSWORD)
+    pro = Process(lims, id=args.pid)
+    pro_udfs = {}
+    samples = {}
+    general_comments = []
+    project_specific_comments = {}
+
+    for name, value in pro.udf.items():
+        pro_udfs[name] = value
+    seq_setup = f"{pro_udfs.get('Read 1 Cycles')}_{pro_udfs.get('Index Read 1')}_{pro_udfs.get('Index Read 2')}_{pro_udfs.get('Read 2 Cycles')}"
+    inst = f"{pro_udfs.get('Instrument')} {pro_udfs.get('Side')}"
+
+    for line in pro_udfs.get("Comment").splitlines():
+        if not line.startswith("//"):
+            if regex_project_line.match(line):
+                result = regex_project_line.search(line)
+                if result.group(1) not in project_specific_comments:
+                    project_specific_comments[result.group(1)] = []
+                project_specific_comments[result.group(1)].append(line)
```

Comment:
I think I understand how these comments are supposed to be written but I think it needs to be documented both in this script and in the LIMS step. I think it can work, but it's perhaps a little brittle when it comes to newlines?

Reply:
Documented in the LIMS step